scripts/inference/mock_setup.py

```diff
@@ -0,0 +1,100 @@
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+import xarray as xr
+from xgboost import XGBRegressor
+
+from gradboost_pv.models.utils import NWP_FPATH, GSP_FPATH
+from gradboost_pv.models.utils import load_nwp_coordinates
+from gradboost_pv.inference.data_feeds import MockDataFeed
+from gradboost_pv.inference.models import (
+    NationalBoostInferenceModel,
+    NationalPVModelConfig,
+    Hour,
+)
+from gradboost_pv.inference.run import (
+    MockDatabaseConnection,
+    NationalBoostModelInference,
+)
+
+MOCK_DATE_RANGE = slice(
+    np.datetime64("2020-08-02T00:00:00"), np.datetime64("2020-08-04T00:00:00")
+)
+DEFAULT_PATH_TO_MOCK_DATABASE = (
+    Path(__file__).parents[2] / "data" / "mock_inference_database.p"
+)
+
+
+def parse_args():
+    parser = ArgumentParser(
+        description="Script to run mock inference of NationalPV model using data from GCP."
+    )
+    parser.add_argument(
+        "--path_to_database",
+        type=str,
+        required=False,
+        default=DEFAULT_PATH_TO_MOCK_DATABASE,
+    )
+    args = parser.parse_args()
+    return args
+
+
+def load_model_from_local(forecast_hour_ahead: Hour) -> XGBRegressor:
+    _model = XGBRegressor()
+    _model.load_model(
+        f"/home/tom/dev/gradboost_pv/data/uk_region_model_step_{forecast_hour_ahead}.model"
```

Comment:
how big is the model, we will need to host it somewhere

Reply:
Checking now, there are also a few other instances of "/home/tom/dev/.." still lying around which will be replaced gradually - like paths to processed NWPs

Reply:
done, will merge this branch and address in separate PR